sklearn/preprocessing/data.py

```diff
@@ -1182,6 +1182,35 @@ def powers_(self):
         return np.vstack(np.bincount(c, minlength=self.n_input_features_)
                          for c in combinations)
 
+    def get_feature_names(self, input_features=None):
```

Comment:
I've been wanting to do it but didn't have time. I'm just writing the book chapter about preprocessing and I'm embarrassed not having that feature ;)